content/300-accelerate/300-caching.mdx

```diff
@@ -108,3 +108,52 @@ For example, consider writing a query on a critical path without explicitly defi
 You must explicitly opt-in to caching if you want to use it. This makes it clear to developers that caching is not enabled by default and helps prevent counterintuitive issues from occurring.
```

Comment:
You must explicitly opt-in to caching for usage. This ensures that developers understand caching is not enabled by default and helps prevent unexpected issues from arising.

@ankur-arch are you running this through the DAGPT? Above is what it gave me when I ran your sentence through

Reply:
Also im unsure what the point of "This ensures that developers understand" is? Are you stating that we did this to make it clear for devs? Or are you saying that devs do this to make it clear to other devs? Not sure if this is needed or it needs to be adjusted to make more sense

Reply:
@mhessdev this was added because a developer asked for caching by default *a long time ago.* I haven't touched that section.